routes/ordersRouter.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express');
+const { getAllOrders, getOrderById, getOrdersByUser } = require('../db/orders');
+const ordersRouter = express.Router();
+
+ordersRouter.get('/', async(req, res, next) => {
```

Comment:
REQUIRED - Should be only sending back if `req.user.isAdmin` (or something)

Reply:
requireAdmin applied.